docs/editor_manual/editing_existing_pages.rst

```diff
@@ -27,3 +28,46 @@ show more information about the page's progress through the workflow, as well as
 If you can perform moderation actions (for example, approval or requesting changes) on the current task, you'll see additional options in the
 action menu at the bottom of the page.
 
+.. _commenting:
+
+Commenting
+__________
+
+Commenting mode is entered using the comment toggle in the bottom right of the header. If there are comments on the page, the number
```

Comment:
Words like "toggle" and maybe even "header" might not mean much to all editors. Also, could we flip the first sentence around?

Something like: "Use the comment icon at the top right to enable commenting mode." ?

Reply:
Thanks, good idea!